### PR TITLE
feat(replacements): add opencost from quay.io to ghcr.io

### PR DESCRIPTION
**Pull request by MindTooth**
## Changes

This replacement will help out with moving OpenCost images from their old registry (quay.io) to their new one (ghcr.io).

Maintainer has given a go ahead [here](https://github.com/opencost/opencost-ui/pull/18#issuecomment-2161607281).

Working pull requests:

- kube-cost-model:  https://github.com/MindTooth/renovate-opencost-test/pull/11
- opencost-ui: https://github.com/MindTooth/renovate-opencost-test/pull/6

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — COMMENTED** [review]
Should we match current version? Otherwise we're potentially upgrading and replacing in one go

**Review by viceice — COMMENTED** [review]
please add match current value or version, so we only move latest good known version.

**Comment by MindTooth** [comment]
I've added an explicit match now.  Thanks, should have been obvious to me.

File: lib/config/presets/internal/replacements.ts [hunk 1]
```diff
@@ -743,6 +744,29 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'opencost-registry-move': {
+    description: 'Replace OpenCost registry from quay.io to ghcr.io.',
+    packageRules: [
+      {
+        description:
+          'Replace `quay.io/kubecost1/kubecost-cost-model` with `ghcr.io/opencost/opencost`.',
+        matchCurrentVersion: '1.108.0',
```
Comment (viceice) [inline lib/config/presets/internal/replacements.ts:753]: is it the last version published there?

Comment (MindTooth) [reply lib/config/presets/internal/replacements.ts:753]: Correct.

That's the version Renovate will provide when using quay.io: https://github.com/MindTooth/renovate-opencost-test/pull/1

Comment (rarkins) [reply lib/config/presets/internal/replacements.ts:753]: And were 109 and 110 skipped?

Comment (MindTooth) [reply lib/config/presets/internal/replacements.ts:753]: They moved over to ghcr.io.  Hence this replacement.

1.110.0 was wanted by the maintainer, but you maybe want the most lowest version first (1.109.0)?

Comment (viceice) [reply lib/config/presets/internal/replacements.ts:753]: ok, use lowest new version as replacement

Comment (MindTooth) [reply lib/config/presets/internal/replacements.ts:753]: Done.

**Review by rarkins — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**